examples/step-31/step-31.cc

```diff
@@ -2202,6 +2213,10 @@ int main (int argc, char *argv[])
       Utilities::MPI::MPI_InitFinalize mpi_initialization (argc, argv,
                                                            numbers::invalid_unsigned_int);
 
+      // This program can only be run in serial. Otherwise, throw an exception.
+      AssertThrow(Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD)==1,
+                  ExcMessage("This program can only be run in serial, use mpirun -np 1 ./step-31"));
```

Comment:
Because if you throw the exception, it means that you are already using mpirun. So just reuse the same command with 1.